Sample/View Controllers/User Groups (Stacked)/StackedProfileViewController.swift

```diff
@@ -89,7 +89,11 @@ class StackedProfileViewController: UIViewController, PanModalPresentable {
 
         roleLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
         roleLabel.topAnchor.constraint(equalTo: nameLabel.bottomAnchor, constant: 4.0).isActive = true
-        bottomLayoutGuide.topAnchor.constraint(greaterThanOrEqualTo: roleLabel.bottomAnchor).isActive = true
+        if #available(iOS 11.0, *) {
+            view.safeAreaLayoutGuide.bottomAnchor.constraint(greaterThanOrEqualTo: roleLabel.bottomAnchor).isActive = true
```

Comment:
Is that correct?
Or you mean `view.safeAreaLayoutGuide.topAnchor` instead of `view.safeAreaLayoutGuide.bottomAnchor`?